test/controllers/users_controller_test.js

```diff
@@ -36,3 +36,7 @@ describe("Endpoints exists", () => {
         });
     });
 });
+
+after(function(){
+    process.exit(0);
+});
```

Comment:
does this exit the entire process? what happens if we have more than one test file